server/app/services/scheduler/filter/volume_plugin.rb

```diff
@@ -19,7 +19,8 @@ def for_service(service, instance_number, nodes)
         }.compact
 
         nodes = nodes.select { |node|
-          (needed_drivers - node.plugins['volume']).empty?
+          volume_drivers = node.volume_drivers.map { |v| v['name'] }
```

Comment:
ATM the `volume_drivers` `name` is the full plugin name with a version tag, like `rexray/s3fs:latest`. This means that creating a volume with `kontena volume create --driver rexray/s3fs` will fail to deploy... the `rexray/s3fs:latest` doesn't match `rexray/s3fs`.